src/components/Dashboard/index.jsx

```diff
@@ -53,14 +84,30 @@ class Dashboard extends React.Component {
     return newArr;
   };
 
+  renderUpsellingStats = partnerStats => () => {
+    const result = this.upSellData(partnerStats);
+    return (
+      <ul className="upSellList">
+        { result.map(stat => (
+          <li key={stat.id} >
```

Comment:
@johnkegz Another eslint for space before the closing bracket

Reply:
@ssewilliam this has been fixed